fahadmak/stackoverflow_clone
# feature(SignUp, Logout): Create signup and logout

**What does this PR do?**
This PR ensures that a user can log in into the application.

**Description of Task to be completed?**
- create UserCreateForm
- create LogoutView, SignUpView
- create User model
- add author field in Question model

**What are the relevant Pivotal Tracker stories?**
[#167227390](https://www.pivotaltracker.com/n/projects/2364284/stories/167227390)

**How should be tested manually?**
Run each of the following commands in your terminal in the respective order.

- `git clone https://github.com/fahadmak/stackoverflow_clone.git`
- `cd stackoverflow_clone`
- `git checkout ft-logout-signup-167227390`
- `virtualenv venv`
- `. venv/bin/activate`
- `pip install -r requirements.txt`
- `run migrations using this command python manage.py migrate`
- `run python manage.py runserver 127.0.0.1:8000`
- `Go to this link in your browser 127.0.0.1:8000`
- `You will be redirected to the login page where you should log in`
- `Click on the 'Register Today' link to go to sign up page`
- `Try to sign up an account and the use it in the login page`

**Screenshots**
<img width="1021" alt="Screen Shot 2019-07-14 at 7 57 49 PM" src="https://user-images.githubusercontent.com/13882936/61186790-014fcb80-a672-11e9-88ed-6366fb71698d.png">
